0004/0004-client-data.avsc

```diff
@@ -22,8 +22,11 @@
         },
         {
             "name":"appVersionName",
-            "type":"string",
-            "doc":"Application version name the data is sent for"
+            "type":[
+                "string",
+                "null"
+            ],
+            "doc":"Application version name the data is sent for. Optional."
```

Comment:
Do not include "Optional."—it is clear from the type field. It is better to describe what `null` means here.

Reply:
Could you help to clarify description? My proposal:
"Application version name the data is sent for. Null in case of information about endpoint belongs to application version does not exist."

Reply:
@rasendubi

Reply:
> It looks like ClientData message should not be changed. I need to change only ExtensionData message.

Agree.

> Could you help to clarify description?

The description looks ok.

Reply:
Thanks.